README.md

```diff
@@ -25,11 +25,7 @@ if err != nil {
 }
 
 
-redisHOST, err := discovery.Host("redis", "tcp")
-if err != nil {
-    logger.Fatal("ERROR: " + err.Error())
-}
-redisPORT, err := discovery.Port("redis", "tcp")
+redisHostPort, err := discovery.HostPort("redis", "tcp")
```

Comment:
Sorry, wasn't arguing - was just asking how you wanted it and was going to do it however that was :)